packages/components/src/radio/tests/chromatic/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,238 @@
+                    <RadioList>
+                        <RadioField>
+                            <Radio value="developer">Developer</Radio>
+                            <Text slot="description">Team Lead</Text>
+                        </RadioField>
+                        <Radio value="designer">Designer</Radio>
+                        <RadioField>
+                            <Radio value="manager">Manager</Radio>
+                            <Text slot="description">Team Manager</Text>
+                        </RadioField>
+                    </RadioList>
+                    <HelperMessage>Select one to continue</HelperMessage>
+                </RadioGroup>
+            </Inline>
+            <h1>Bordered & Invalid</h1>
+            <Inline alignY="start">
+                <RadioGroup {...props} size="sm" variant="bordered" isInvalid>
+                    <Label>Roles</Label>
+                    <RadioList>
+                        <Radio value="developer">Developer</Radio>
+                        <Radio value="designer">Designer</Radio>
+                        <Radio value="manager">Manager</Radio>
+                    </RadioList>
+                    <ErrorMessage>This field is required</ErrorMessage>
+                </RadioGroup>
+                <RadioGroup {...props} size="md" variant="bordered" isInvalid>
+                    <Label>Roles</Label>
+                    <RadioList>
+                        <Radio value="developer">Developer</Radio>
+                        <Radio value="designer">Designer</Radio>
+                        <Radio value="manager">Manager</Radio>
+                    </RadioList>
+                    <ErrorMessage>This field is required</ErrorMessage>
+                </RadioGroup>
+            </Inline>
+            <h1>Styling</h1>
+            <Inline>
+                <RadioGroup border="warning-strong">
+                    <Label>Numbers</Label>
+                    <RadioList>
+                        <Radio value="1">1</Radio>
+                        <Radio value="2">2</Radio>
+                        <Radio value="3">3</Radio>
+                    </RadioList>
+                </RadioGroup>
+                <RadioGroup className="bg-red">
+                    <Label>Numbers</Label>
+                    <RadioList>
+                        <Radio value="1">1</Radio>
+                        <Radio value="2">2</Radio>
+                        <Radio value="3">3</Radio>
+                    </RadioList>
+                </RadioGroup>
+                <RadioGroup style={{ backgroundColor: "red" }}>
+                    <Label>Numbers</Label>
+                    <RadioList>
+                        <Radio value="1">1</Radio>
+                        <Radio value="2">2</Radio>
+                        <Radio value="3">3</Radio>
+                    </RadioList>
+                </RadioGroup>
+            </Inline>
+            <h1>Zoom</h1>
+            <Inline alignY="start">
```

Comment:
for some reason, in the zoom tests i can't select a value

Reply:
we decided to ignore this for now, since css zoom is known to cause issues with events. The browser zoom did not cause an issue.